Sources/LambdaConverters.Wpf/Properties/AssemblyInfo.cs

```diff
@@ -4,12 +4,6 @@
 // General Information about an assembly is controlled through the following 
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("LambdaConverters.Wpf")]
-[assembly: AssemblyDescription("")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("LambdaConverters")]
-[assembly: AssemblyCopyright("© 2017 Michael Damatov, Dimitri Enns.")]
```

Comment:
You can include this information in the .csproj file.

https://stackoverflow.com/a/42143079/24874

I'm not familiar with the deployment project here, but on the projects I've converted to this new csproj format, I've been able to remove the `.nuspec` file, and consolidate that information here too.